src/main/java/com/wolox/training/controllers/UserController.java

```diff
@@ -122,4 +124,11 @@ public User removeBookToUser(@PathVariable Long userId, @PathVariable Long bookI
         user.removeBook(book);
         return userRepository.save(user);
     }
+
+    @GetMapping("/username")
```

Comment:
just as a  general comment, some other common names for this endpoint are `/me`, `/user`, etc 😄